### DISCUSSION TITLE
supermarket tab in shopping list v2

### DISCUSSION THREAD

**Discussion by TWART016** [TandoorRecipes/recipes#1236] [Ideas]
In shopping list v2 are all supermarkets and shopping categories listed in tab supermarkets. Does it make more sense if there are just the supermarkets I need ingredients for my recipes?

Therefore I can better see which supermarkets I still have to go to.

**Comment by smilerz** [comment]
That tab is for managing supermarkets and categories.

**Reply by TWART016** [reply]
That can be also done in `/list/supermarket/` and `/list/supermarket-category/`.

Does it make sense to a list to which supermarkets I need to go?

In `shopping list / supermarkets` I can click on edit Supermarkets. Then I can change the order of the shopping categories on the right site. I would have expected to be able to change the Supermarkets names or the Supermarkets <> Categories mapping but not the order of the categories.

**Comment by smilerz** [comment]
It’s on the shopping page so that you can quickly make changes during a shopping trip. 

Foods are not ties to a single supermarket, so a list of supermarkets doesn’t make much sense.

**Comment by TWART016** [comment]
In Supermarket tab I just can asign a category to a supermarket. I don't think that is needed very often. With 2 clicks I can open supermarket list `list/supermarket/` and do the same. 

Too bad that food can not be tied to supermarkets. Would make more sense in my eyes than categories, but that's another topic.

**Comment by sortab** [comment]
Is there a way to have a "permanent" shopping list for each supermarket I define? Or define some foods that are only/preferably bought in a specific supermarket?
E.g. one supermarket has a great salad dressing which is specific for that supermarket, and another supermarket has a great breakfast cereal that is only sold in that specific supermarket.

**Comment by smilerz** [comment]
You can assign categories to supermarkets and food to supermarkets.  
You can filter a supermarket to just its assigned categories.

If you only buy tomatoes only from Supermarket X and onions only from Supermarket Y you can create a produce-x and produce-y to accomplish what you want.

**Reply by sortab** [reply]
Thanks again for the help. I'll look further into the categories and shopping lists.
